skillful_network_server/src/main/java/fr/uca/cdr/skillful_network/controller/user/FollowStateTrackerController.java

```diff
@@ -37,38 +39,93 @@
     }
 
     @PreAuthorize("hasAnyRole('ENTREPRISE','ORGANISME','USER')")
-    @GetMapping(value = "/all/{Id}")
+    @GetMapping(value = "/{Id}")
     public ResponseEntity<FollowStateTracker> getFSTById(@PathVariable(value = "Id") Long Id) {
```

Comment:
Les paramètres du méthodes devraient toujours commencer par une minuscule. Je dirais la même chose pour le paramètre du mapping alors.

Reply:
yup .. new commit is coming